openapi/pagerduty/Dependecies.toml

```diff
@@ -0,0 +1,11 @@
+[[dependency]]
+org = "ballerina"
+name = "http"
+version = "1.1.0-beta.2"
+
+[[dependency]]
+org = "ballerina"
+name = "url"
+version = "1.1.0-beta.2"
```

Comment:
Both these versions have to be changed to `2.0.0`

Reply:
Yeah. Let's change this.
/Thanks